### PR TITLE
Adding DDOutlier R package from CRAN

### PR DESCRIPTION
**Pull request by jackfeltham**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jackfeltham** [comment]
@conda-forge-admin, please rerun bot

(Checking that Windows install failure wasn't sporadic)

**Comment by jackfeltham** [comment]
@conda-forge-admin, please ping team

I am getting conflicts in the CI check for the Windows installation of this CRAN package. (They seem to be related to conflicting conda epochs but I don't have much experience submitting packages to conda so I'm not sure.). the Linux and Mac CI checks seem to work normally.

Thanks in advance for any help you can give me!

**Comment by jackfeltham** [comment]
@conda-forge/help-r

Pinging the relevant team.

A copy of my message is below:

> I am getting conflicts in the CI check for the Windows installation of this CRAN package. (They seem to be related to conflicting
> conda epochs but I don't have much experience submitting packages to conda so I'm not sure.). the Linux and Mac CI checks
> seem to work normally.
>
> Thanks in advance for any help you can give me!

File: recipes/r-ddoutlier/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <doi:10.1016/j.neucom.2017.02.039> and Gao, J., Hu, W., Zhang, X. & Wu, Ou. (2011)
+    <doi:10.1007/978-3-642-20847-8_23>."
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - jackfeltham
```
Comment (mfansler) [inline recipes/r-ddoutlier/meta.yaml:72]: (optional) Please retain the Conda Forge R Team as maintainers:

```suggestion
    - conda-forge/r
    - jackfeltham
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Everything looks good - thanks for submitting this! ❤️ 

Couple notes:
- Would you mind adding back the R team? This may trigger a linting error, which can be safely ignored.
- As this is a `noarch` build, the Windows build fail can be ignored (though for future reference [it is a known bug](https://github.com/conda-forge/staged-recipes/issues/25319)).

**Comment by jackfeltham** [comment]
> Everything looks good - thanks for submitting this! ❤️
> 
> Couple notes:
> 
> * Would you mind adding back the R team? This may trigger a linting error, which can be safely ignored.
> * As this is a `noarch` build, the Windows build fail can be ignored (though for future reference [it is a known bug](https://github.com/conda-forge/staged-recipes/issues/25319)).

I've just added them back. I only removed them initially because of the linting error so if that doesn't matter then I'm more than happy for them to be on there. :-)

Are there any other amends for me to add or are we good to go?

**Comment by jackfeltham** [comment]
@conda-forge/help-r @mfansler 

Just pinging in case it's standard procedure to do so whenever changes are made (sorry to hassle you if not.)

**Review by mfansler — APPROVED** [review]

**Pull request merged**